core/src/com/gaalf/game/precreatedEntities/balls/BallFactory.java

```diff
@@ -23,10 +22,14 @@ public BallFactory(World world, GameAssetManager assetManager){
     @Override
     public Entity createEntity(String type, String playerName, int playerNumber, TiledMap tiledMap) {
         switch (type) {
-            case "golfBall":
+            case "Golfball":
```

Comment:
why change? I think all the string values was better as they were

Reply:
The texture atlas file has these values. By naming them with capital letters we can show the texture atlas values directly in the application as opposed to giving each and every one custom properties.

Reply:
I see. Make sense. But isn't golf ball for instance written in two words?

Reply:
Come on, give a guy a break!!